src/core/public/public.api.md

```diff
@@ -778,9 +779,15 @@ export interface NavigateToAppOptions {
     openInNewTab?: boolean;
     path?: string;
     replace?: boolean;
+    skipAppLeave?: boolean;
     state?: unknown;
 }
 
+// @public
+export interface NavigateToUrlOptions {
```

Comment:
New options param to bucket any navigation options for the method into. For now, it only contains the `skipAppLeave` flag.